assets/scss/modules/_tables.scss

```diff
@@ -39,21 +39,55 @@
 
 .dp-library {
   @mixin dp-c-table($bg-color-zebra, $font-size-table, $bg-color-hover) {
+    .dp-expand-results {
+      position: relative;
+      padding: $dp-spaces--lv1 $dp-spaces--lv3 $dp-spaces--lv0 $dp-spaces--lv0;
+
+      .icon-arrow-next {
+        position: absolute;
+        transform: rotate(0deg);
+      }
+
+      &.dp-rgpd-open-results {
+        .icon-arrow-next {
+          position: absolute;
+          transform: rotate(90deg);
+        }
+      }
+    }
+
     .dp-c-table {
       width: 100%;
       font-size: $font-size-table;
       border-collapse: collapse;
       border-spacing: $dp-spaces--lv0;
       background: $dp-color-white;
 
+      &.dp-rgpd-table {
```

Comment:
Perhaps instead of calling this table `dp-rgpd-table` you could call this by a more meaningful name like `dp-composed-table`, or some other name that indicates a complex and composited table. Think that we may use this in another table.